style.scss

```diff
@@ -0,0 +1,312 @@
+:root {
+	--light-color: rgb(255, 255, 255);
+	--dark-color: rgb(51, 51, 51);
+	--blue: rgb(10, 188, 235);
+	--dark-grey: rgb(70, 70, 70);
+	--green: rgb(37, 241, 139);
+}
+* {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+html {
+	font-size: 62.5%;
+	scroll-behavior: smooth;
+}
+body {
+	font-family: 'Montserrat', sans-serif;
+	height: 100%;
+	font-size: 15px;
+	background-color: #fff;
+}
+.wrapper {
+	margin: 0 auto;
+	max-width: 1200px;
+	width: 100%;
+}
+.background {
+	background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+}
+.nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	&__logo {
+		padding: 2rem 1rem;
+	}
+	&__menu {
+		padding: 2rem;
+		&--list {
+			font-size: 2rem;
+			list-style: none;
+		}
+		&--item {
+			display: inline-block;
+			padding: 0 2rem;
+			a {
+				font-weight: 600;
+				color: var(--light-color);
+				text-decoration: none;
+				transition: color 0.5s;
+			}
+			a:hover {
+				color: var(--dark-color);
+			}
+		}
+	}
+}
+.header {
+	display: flex;
```

Comment:
ok, teraz widzę, że jest to automatycznie dodawane, ale pamiętaj, że to dodatkowa ilość kodu, która nie zawsze jest nam potrzebna ;)

Reply:
no tak ale ale to już wtyczka generuje automatycznie. Pisze w metodologii bem i łącze to sassem żeby przyspieszyć swoją prace. ale jak zauważyłeś to jeszcze muszę to poprawić.